content/md/en/docs/fundamentals/rust-basics.md

```diff
@@ -36,7 +36,7 @@ Substrate has a lot of its of own primitive types and associated traits that mak
 ## Macros
 
 When you learn about writing FRAME pallets, you will quickly come across a variety of different types of macros designed to abstract and enforce any runtime specific requirements.
-With them, you're able to focus on writing idiomatic Rust, minimizing the overhead of writing extra code that you would otherwise need to write code that correctly interacts with a runtime.
+With them, you're able to focus on writing idiomatic Rust, minimizing the overhead of writing extra code that you would otherwise need to write that correctly interacts with a runtime.
```

Comment:
```suggestion
These macros allow you to focus on writing idiomatic Rust and application-specific logic instead of the common code required to interact with a runtime.
```